packages/shared/src/components/KeyboardShortcutLabel.tsx

```diff
@@ -10,6 +10,19 @@ export const KeyboadShortcutLabel = ({
   className,
   keys,
 }: KeyboadShortcutLabelProps): ReactElement => {
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    // keyboard shortcut buttons are most of the times dependant
+    // on browser features so we don't render them in SSR to
+    // avoid hydration mismatches
```

Comment:
Not for now, but should we maybe add a backlog ticket to make this into it's own hook for clarity down the line?

Reply:
Feel like hook would hide it and make it more magical and then in hindsight used whenever someone encountered any SSR issue vs most of the time you can solve it by other mechanisms. Agree we should keep it in check this is why I added comment for now to explain.

Reply:
Yeah comment is perfect for now :)